nilearn/image/resampling.py

```diff
@@ -300,8 +297,7 @@ def _resample_one_img(
 
     # Suppresses warnings in https://github.com/nilearn/nilearn/issues/1363
     with warnings.catch_warnings():
-        if _compare_version(scipy.__version__, ">=", "0.18"):
-            warnings.simplefilter("ignore", UserWarning)
+        warnings.simplefilter("ignore", UserWarning)
```

Comment:
Is it still relevant to suppress these warnings? Do they still occur in scipy>=1.6.0

Reply:
let me check

Reply:
warnings are still there, so it does make sense to suppress the warning but I will add a comment because without the scipy reference in the if block if becomes unclear why we are suppressing it

Reply:
actually improved the filtering instead